samples/snippets/create_lite_pubsub_export_subscription_example.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python
+
+# Copyright 2022 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This application demonstrates how to create a subscription that writes
+messages to a Pub/Sub topic. For more information, see documentation at
+https://cloud.google.com/pubsub/lite/docs/export-subscriptions.
```

Comment:
Do we know when this page will be up?

Reply:
The page will be up in mid December. I'll update this link if the page name changes.